epytope/test/TestEpitopePrediction.py

```diff
@@ -50,7 +51,19 @@ def test_single_peptide_input_mhcII(self):
                 if not isinstance(model, AExternalEpitopePrediction):
                     if all(a in model.supportedAlleles for a in self.mhcII):
                         res = model.predict(self.peptides_mhcII, alleles=self.mhcII)
+    
+    def test_prediction_for_mouse(self):
+        syfpeithi = EpitopePredictorFactory("Syfpeithi")
+        res = syfpeithi.predict(self.peptides_mhcI, alleles=self.mouse)
 
+        mhcflurry = EpitopePredictorFactory("mhcflurry")
+        mhcnuggets = EpitopePredictorFactory("mhcnuggets-class-1")
+        netmhc = EpitopePredictorFactory("netmhc")
```

Comment:
I think these 2 need to go the external tests right?

Reply:
Since we do not execute the tools in there I think it is fine to have these tests in there.